app/models/customer.py

```diff
@@ -0,0 +1,24 @@
+from flask import current_app
+from app import db
+from flask_sqlalchemy import SQLAlchemy
+
+class Customer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String)
+    postal_code = db.Column(db.String)
+    phone = db.Column(db.String)
+    registered_at = db.Column(db.DateTime)
+    #set videos_checked_out_count equal to len(videos)??
+    videos_checked_out_count = db.Column(db.Integer)
+    videos = db.relationship("Rental", back_populates="customer")
```

Comment:
This would really make more sense as a rentals relationship instead of videos.

```suggestion
     rentals = db.relationship('Rental',backref = 'customer')
```